src/test/rules/handleCallbackErrRuleTests.ts

```diff
@@ -86,7 +91,35 @@ ruleTester.addTestGroup('custom-error-name-pass', 'should pass with custom error
 ]);
 
 ruleTester.addTestGroup('custom-error-name-fail', 'should fail with custom error name', [
-  { code: `function(errorMsg, stream) { }`, options: ['errorMsg'], errors: [error] }
+  { code: `function(errorMsg, stream) { }`, options: ['errorMsg'], errors: [error] },
+  {
+    code: `error => console.error('Could not print the document');`,
+    options: ['error'],
+    errors: [error]
+  },
+  {
+    code: `error => something.something.error.something.err('Could not print the document');`,
+    options: ['^(err|error)$'],
+    errors: [error]
+  },
+  {
+    code: `error => console.error(error.stack);`,
+    options: ['^(err|error)$']
+  },
+  {
+    code: `error => console.error(error.stack);`,
+    options: ['^(err|error)$', { allowProperties: false }],
+    errors: [strictError]
+  },
+  {
+    code: 'var test = err => err.message;',
+    options: ['err', { allowProperties: false }],
+    errors: [strictError]
+  },
+  {
+    code: `error => error ? console.error(error.stack) : console.log('no error');`,
```

Comment:
I think this should also pass since if `error` doesn't exist, it has been handled, and if it exists, it is handled fine since allowProperties is not set to false.